.github/workflows/e2e.yaml

```diff
@@ -128,3 +134,41 @@ jobs:
         run: |
           cd ${{ matrix.usecase }}
           go test -v --tags=e2e
+
+  eec-confmap-provider:
+    runs-on: ubuntu-latest
+    needs: docker-build
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "~1.21.3"
+          check-latest: true
+          cache-dependency-path: internal/tools/go.sum
+
+      - name: Cache tools
+        uses: actions/cache@v4
+        with:
+          path: .tools
+          key: ci-tools-${{ runner.os }}-${{ hashFiles('internal/tools/go.sum') }}
+
+      - name: Touch tools
+        run: |
+          mkdir -p .tools
+          touch .tools/*
+
+      - name: Download artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: dynatrace-otel-collector-bin
+          path: bin
+
+      - run: chmod +x bin/*
+
+      - name: Run e2e tests
+        run: |
+          cd confmap/provider/eecprovider
+          go test -v --tags=e2e
```

Comment:
are unit tests ran as part of this as well? Or are they part of a different workflow?

Reply:
Unit tests are run as part of our main CI workflow, specifically at this step: https://github.com/Dynatrace/dynatrace-otel-collector/blob/main/.github/workflows/ci.yaml#L43.